docs/extending/extending_draftail.md

```diff
@@ -83,6 +95,7 @@ Blocks are nearly as simple as inline styles:
 ```python
 import wagtail.admin.rich_text.editors.draftail.features as draftail_features
 from wagtail.admin.rich_text.converters.html_to_contentstate import BlockElementHandler
+from wagtail import hooks
```

Comment:
Please pull out these documentation changes for additional hooks imports, these can be done in a separate PR.

Reply:
I will work on this.